homeassistant/components/emulated_roku/config_flow.py

```diff
@@ -0,0 +1,62 @@
+"""Config flow to configure emulated_roku component."""
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME
+from homeassistant.core import callback
+from homeassistant.util import slugify
+from .const import (DOMAIN, CONF_LISTEN_PORT, CONF_HOST_IP,
+                    CONF_ADVERTISE_IP, CONF_ADVERTISE_PORT,
+                    CONF_UPNP_BIND_MULTICAST)
+
+
+@callback
+def configured_servers(hass):
+    """Return a set of the configured servers."""
+    return set(slugify(entry.data[CONF_NAME]) for entry
+               in hass.config_entries.async_entries(DOMAIN))
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class EmulatedRokuFlowHandler(config_entries.ConfigFlow):
+    """Handle an emulated_roku config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
+
+    _hassio_discovery = None
+
+    def __init__(self):
+        """Initialize emulated_roku configuration flow."""
+        pass
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        return await self.async_step_init(user_input)
+
+    async def async_step_init(self, user_input=None):
```

Comment:
You don't need step init, just replace it with step user

Reply:
Thanks. Seen the docs, however I could not find any components that didn't use it. Thought it might be needed for compatibility.